doc/user/error-interaction.rst

```diff
@@ -158,3 +158,15 @@ You can put errors into the kill ring with `C-c ! w`:
                 M-0 M-x flycheck-copy-errors-as-kill
 
    Like `C-c ! w` but do not copy the error messages but only the error IDs.
+
+Explain errors
```

Comment:
Can you move the section before the "kill errors" section?  I think "explaining" errors is the much more important use case as it offers immediate insight as to the cause of an error, whereas killing the error ID/message for use with Google is only the last resort, really.